versions.mk

```diff
@@ -16,6 +16,7 @@ MDBOOK_VERSION := v0.4.5
 PLANTUML_VERSION := 1.2020.16
 GH_VERSION := 2.7.0
 CERT_MANAGER_VERSION := v1.10.1
+CAPI_VERSION := v1.4.0
 CAPI_VERSION := v1.3.5
```

Comment:
do we need to remove line: CAPI_VERSION := v1.3.5

Reply:
thanks for noticing, I will remove

Reply:
Done